src/sound.c

```diff
@@ -297,67 +293,67 @@ sound_free(void)
 	vim_free(scb);
     }
 }
-# endif
+#  endif
 
-#elif defined(MSWIN)
+# elif defined(MSWIN)
 
 /*
  * Sound implementation for MS-Windows.
  */
 
 static HWND g_hWndSound = NULL;
 
-    static LRESULT CALLBACK
+static LRESULT CALLBACK
 sound_wndproc(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
 {
-    soundcb_T	*p;
+    soundcb_T *p;
 
     switch (message)
     {
-	case MM_MCINOTIFY:
-	    for (p = first_callback; p != NULL; p = p->snd_next)
-		if (p->snd_device_id == (MCIDEVICEID) lParam)
-		{
-		    char	buf[32];
+    case MM_MCINOTIFY:
```

Comment:
The "case" statement should be indented one 'shiftwidth' from the "switch" statement.